gui/src/main/java/org/jboss/as/console/client/rbac/SecurityFrameworkImpl.java

```diff
@@ -100,7 +100,23 @@ public String filter(String key) {
 
                     @Override
                     public String[] filterTuple(String key) {
-                        return null;
+                        switch (key) {
```

Comment:
Ok, this is obviously a misunderstanding on my part.  The reason I modified this method was because filter(String key) always returns the same address value when "addressable.group" is the key.  Whereas, I required all of the server-group addresses in order to issue the appropriate request to EAP.  My thinking was that I would return a tuple of all the matching server-groups.  

What do you think would be a better alternative to filterTuple()?